sql/src/test/quidem/qatests/kttm_nested/ingest.json

```diff
@@ -0,0 +1,73 @@
+{
+  "type": "index_parallel",
+  "spec": {
+    "ioConfig": {
+      "type": "index_parallel",
+      "inputSource" : {
+        "type" : "local",
+        "baseDir" : "sql/src/test/quidem/qatests/kttm_nested/data",
+        "filter" : "kttm-nested-v2-2019-08-25.json.gz"
```

Comment:
we should find a way to access the already accessible `kttm-nested-v2-2019-08-25.json` on the classpath; and that should be the way to get larger datafiles

Reply:
What is the equivalent of `!use druidtest:///?componentSupplier=KttmNestedComponentSupplier` if to use this dataset for `AllDruidEnginesComponentSupplier`, `DartComponentSupplier`, `StandardMSQComponentSupplier`, and `StandardComponentSupplier`?  These tests originally used `!use druidtest:///?componentSupplier=KttmNestedComponentSupplier`.  We need to make them run for all 3 engines, but I couldn't find a way to do so.  That's why the dataset was added.

Reply:
Will change the tests to use smaller data files instead.

Reply:
Addressed in https://github.com/apache/druid/pull/17861/commits/e2df8ea12b4b27df86b5fa28668eb38f738cc679